lib/find.coffee

```diff
@@ -117,6 +117,20 @@ module.exports =
       'find-and-replace:select-skip': (event) ->
         selectNextObjectForEditorElement(this).skipCurrentSelection()
 
+    maybeRegisterToAutocompletePlus = (atomPackage) =>
```

Comment:
I'm mainly concerned about interacting with the package registry in this way. Services are a cleaner way of interacting with another package and are also semantically versioned, which is important as APIs evolve.

But I think I'd prefer to make the change in `autocomplete-plus` directly. It seems simpler on balance. I wonder if you could craft a generic class like `provide-default-autocompletions` or something that anyone could apply to an editor they register?